gradle/libs.versions.toml

```diff
@@ -52,15 +56,3 @@ powerMock = [
     "powerMock-apiMockito2",
     "powerMock-moduleJunit4",
 ]
-
-[plugins]
```

Comment:
I'd add a comment why `[plugins]` section is missing despite our use of multiple Gradle plugins in the project, as it's somewhat non-trivial